src/App.css

```diff
@@ -1,39 +1,57 @@
-.App {
-  text-align: center;
+* {
```

Comment:
| Your Design | Wireframe |
| --- | --- |
| ![image](https://github.com/CesarHerr/math_magicians/assets/93039359/5cb5626c-4fee-410c-95ce-5a106ebff0e0) | ![image](https://github.com/CesarHerr/math_magicians/assets/93039359/adaae3f4-e39c-4945-916f-8044faaad9e3) |

**[OPTIONAL]**

- [ ] It would be nice if you follow the design and remove the border around the calculator to make your calculator look similar to the design which is more professional :+1:

Reply:
@HAMMAS-SALEEM Thanks for you review, I'll take your suggestion 👍